autoload/sl.vim

```diff
@@ -78,6 +78,10 @@ function! sl#animate()
   let l:wrap_backup = &l:wrap
   setlocal nowrap
   redraw
+  if exists('*sound_playfile')
+    let l:filepath = expand('<sfile>:h:h') . '/sl.wav'
+    call sound_playfile(filepath)
```

Comment:
Sorry, nit pick. `l:filepath`

Reply:
And I'm sorry many times to take a mistake.